Documentation/ExtensionArchitecture/HowTo/UpdateExtensions/UpdateWizards/Creation.rst

```diff
@@ -42,6 +42,9 @@ The registration of an upgrade wizard is done directly in the class by adding
 the class attribute :php:`\TYPO3\CMS\Install\Attribute\UpgradeWizard`. The
 :ref:`unique identifier <upgrade-wizards-identifier>` is passed as an argument.
 
+Your extension must have a :ref:`Services.yaml <t3coreapi:extension-configuration-services-yaml>`
+with `autowire = true` for the upgrade wizard to be detected.
```

Comment:
Actually, it *should* also work with autowire=false and tagging manually in Services.yaml:

    services:
       YourVender\YourExtension\UpgradeWizard\YourUpgradeWizard:
          tags:
             - name: install.upgradewizard
               identifier: your-upgrade-wizard

Reply:
And: Do you mean autowire or autoconfigure? I think, this should be autoconfigure=true without tagging manually.

Reply:
Yup, should be autoconfigure.

Autowiring:
"... reads the type-hints on your constructor (or other methods) and automatically passes the correct services to each method."
https://symfony.com/doc/current/service_container/autowiring.html

Reply:
So, wie should add something like this:
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Backend/ContextualMenu.html#step-3-registration

Reply:
@brotkrueml would you like to take over? It seems like I am not deep enough into this, just noted that without a services.yaml it deosnt work

Reply:
Done